packages/cozy-pouch-link/src/jsonapi.js

```diff
@@ -1,6 +1,7 @@
 export const normalizeDoc = (doc, doctype) => {
   const id = doc._id || doc.id
-  return { id, _id: id, _type: doctype, ...doc }
+  const rev = doc._rev || doc.rev
```

Comment:
When you update a document pouch response include `rev` but the document have already old `_rev` so it's indispensable to change it:
```suggestion
  const rev = doc.rev || doc._rev
```

Reply:
Thanks !